resources/translations/fi.edn

```diff
@@ -79,6 +80,7 @@
                             :resource-in-use "Resurssi on käytössä:"
                             :workflow-archived "Työvuo on arkistoitu:"
                             :workflow-in-use "Työvuo on käytössä:"}
+                   :export-applications "Vie hakemuksia"
```

Comment:
Vie sounds weid but I guess it is the typical translation for export

Reply:
I double-checked and it indeed seems to be typical translation, see, e.g., https://docs.gimp.org/fi/gimp-file-export.html

I think the slight problem here is that there is less context for the verb "Vie" in the UI, compared to, e.g., a file menu containing a similar item "Vie..." where it is easier to interpret correctly.

Another option could be "Muunna hakemuksia", but the meaning would then be slightly different from "Export".